src/system_tray.cpp

```diff
@@ -40,11 +40,13 @@
   #include "main.h"
   #include "platform/common.h"
   #include "process.h"
+  #include "version.h"
 
 using namespace std::literals;
 
 // system_tray namespace
 namespace system_tray {
+  const char* tooltipText = PROJECT_NAME;
```

Comment:
You should be able to just assign directly to `tray.tooltip` without needing a separate variable.